backend/test/flights/unit/crud.test.ts

```diff
@@ -28,8 +30,10 @@ describe('/flights/', () => {
       .post('/flights/')
       .send({
         _id: '1',
-        departure_city: 'Jakarta Bali',
-        arrival_city: 'Temburong Thailand',
+        departure_city: 'Jakarta',
+        departure_country: 'Bali',
+        arrival_city: 'Temburong',
+        arrival_country: 'Brunei',
         departure_date: '2022-04-28',
         departure_time: '7:38 AM',
```

Comment:
Updated sql for flights created issue here, I have made some updates to the test to make it work. Do advise if it is okay

Reply:
thanks, looks good.